src/components/Add.js

```diff
@@ -0,0 +1,54 @@
+import { useDispatch } from 'react-redux';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please don't disable eslint rules. There are here to help us write best code possible.